examples/go/hello-lambda/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"embed"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/awslabs/aws-lambda-go-api-proxy/httpadapter"
+
+	"github.com/mattermost/mattermost-plugin-apps/apps"
+	"github.com/mattermost/mattermost-plugin-apps/apps/mmclient"
+)
+
+//go:embed manifest-http.json
+var manifestHTTPData []byte
+
+//go:embed manifest.json
+var manifestAWSData []byte
+
+//go:embed pong.json
+var pongData []byte
+
+//go:embed bindings.json
+var bindingsData []byte
+
+//go:embed send_form.json
+var formData []byte
+
+//go:embed static
+var static embed.FS
+
+func main() {
+	localMode := os.Getenv("LOCAL") == "true"
+
+	// Serve its own manifest as HTTP for convenience in dev. mode.
+	if localMode {
+		http.HandleFunc("/manifest.json", writeJSON(manifestHTTPData))
+	} else {
+		http.HandleFunc("/manifest.json", writeJSON(manifestAWSData))
+	}
```

Comment:
1/5 that it's idiomatic to define the manifest as a go variable and rewrite based on the mode to keep things DRY.